packages/@aws-cdk/aws-appmesh/lib/client-policy.ts

```diff
@@ -0,0 +1,110 @@
+import * as acmpca from '@aws-cdk/aws-acmpca';
+import * as cdk from '@aws-cdk/core';
+import { CfnVirtualNode } from './appmesh.generated';
+
+enum CertificateType {
+  ACMPCA = 'acm',
+  FILE = 'file'
```

Comment:
Trailing comma here:

```suggestion
  FILE = 'file',
```